doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -38,9 +38,11 @@ They return a structure that contains the different file names::
     >>> # The different files
     >>> print(list(haxby_files.keys()))
     ['mask_house_little', 'anat', 'mask_house', 'mask_face', 'func', 'session_target', 'mask_vt', 'mask_face_little']
-    >>> #  Path to first functional file
+    >>> # Path to first functional file
     >>> print(haxby_files.func[0] # doctest: +ELLIPSIS)
     /.../nilearn_data/haxby2001/subj1/bold.nii.gz
+    >>> # Provide information on the dataset
+    >>> print(haxby_files.description)
```

Comment:
It'd be helpful to at least put the beginning of the description with an ellipsis to give a feeling what the output of this line should be